src/sif/autocost.cc

```diff
@@ -232,7 +232,8 @@ class AutoCost : public DynamicCost {
   virtual Cost EdgeCost(const baldr::DirectedEdge* edge,
                         const graph_tile_ptr& tile,
                         const uint32_t seconds,
-                        uint8_t& flow_sources) const override;
+                        uint8_t& flow_sources,
+                        const uint64_t seconds_from_now) const override;
```

Comment:
Can we pass one parameter `time_info` instead of `seconds` and `seconds_from_now` ?

Reply:
for sure, that was what i was originally planning to do, encapsulate all the timing information into one thing that we coudl pass around. its great to finally do this last part

Reply:
Moved to time_info for EdgeCost function. @genadz, @kevinkreiser could you please check the latest commit? Not 100% sure if setting invariant time info is a good option everywhere.